articles/quickstart/backend/aspnet-core-webapi/03-troubleshooting.md

```diff
@@ -91,7 +91,7 @@ To debug potential configuration issues, inspect the log files for your applicat
 
 In this example, we run the application from the command line and inspect the console log output.
 
-### 1. Are you passing the JWT in the Authorization header?
+### 1. Are you Passing the JWT in the Authorization Header?
```

Comment:
I know this wasn't part of your diff but the heading levels should be consistent here; this first point is a level three but the others below are level 2's. It looks to me like they should ideally be level 3s.

Also, if we're changing them to  _level 3_ headings, we can go ahead and leave the casing the same as it was before (sentence case).